src/_http/index.js

```diff
@@ -118,25 +118,30 @@ function secretHeader(secret) {
 
 /** @ignore */
 function getDefaultHeaders() {
-  var headers = {
-    'X-Fauna-Driver': 'Javascript',
-    'X-FaunaDB-API-Version': packageJson.apiVersion,
+  var driverEnv = {
+    driverVersion: packageJson.version,
   }
 
   if (util.isNodeEnv()) {
-    // TODO: should be at the browser as well. waiting to enable CORS headers
-    headers['X-Fauna-Driver-Version'] = packageJson.version
-    headers['X-Runtime-Environment'] = getNodeRuntimeEnv()
-    headers['X-Runtime-Environment-OS'] = require('os').platform()
-    headers['X-NodeJS-Version'] = process.version
+    driverEnv.driver = 'nodejs'
+    driverEnv.languageVersion = process.version
+    driverEnv.env = getNodeRuntimeEnv()
+    driverEnv.os = require('os').platform()
   } else {
-    // TODO: uncomment when CORS enabled
-    // var browser = require('browser-detect')()
-    // headers['X-Runtime-Environment'] = browser.name
-    // headers['X-Runtime-Environment-Version'] = browser.version
-    // headers['X-Runtime-Environment-OS'] = browser.os
+    driverEnv.driver = 'javascript'
+    driverEnv.env = window.navigator ? window.navigator.userAgent : 'unknown'
+  }
+
+  var headers = {
+    'X-FaunaDB-API-Version': packageJson.apiVersion,
   }
 
+  // TODO: api cors must be enabled to accept header X-Driver-Env
+  if (util.isNodeEnv()) {
+    headers['X-Driver-Env'] = Object.keys(driverEnv)
+      .map(key => [key, driverEnv[key].toLowerCase()].join('='))
+      .join('; ')
```

Comment:
Are we still supporting ECMA 5? If so, is `.join` compatible?

Reply:
this is a standard from the very first ECMA https://www.ecma-international.org/wp-content/uploads/ECMA-262_1st_edition_june_1997.pdf